MANUAL.md

```diff
@@ -29,6 +29,36 @@ Cloud Foundry.
     assign a pushed application that did not specify its memory
     requirements, explicitly.  Defaults to `256`.
 
+  - `default_app_disk_in_mb` - How much disk (in megabytes) to
+    assign a pushed application that did not specify its memory
+    requirements, explicitly. Defaults to `1000`.
+
+  - `uaa_lockout_failure_count` - Amount of failed UAA login attempts
+    before lockout.
+
+  - `uaa_lockout_failure_time_between_failures` - How much time
+    (in seconds) in which `uaa_lockout_failure_count` must occur in 
+    order for account to be locked. Defaults to `1200`.
+  
+  - `uaa_lockout_punishment_time` - How long (in seconds) the account
+    is locked out for violating `uaa_lock_failure_count` within 
+    `uaa_lockout_failure_time_between_failures`. Defaults to `300`.
```

Comment:
Punishment seems a bit harsh.

Perhaps `uaa_lockout_time`?

Reply:
Also the parameter is `uaa_lockout_time_between_failures`, but if were changing the name anyways.